src/lib/optimizer/strategy/in_expression_rewrite_rule.cpp

```diff
@@ -43,6 +44,10 @@ void rewrite_to_join(const std::shared_ptr<AbstractLQPNode>& node,
     list_as_table->append_chunk({value_segment});
   });
 
+  // All regular tables have statistics. Even if the JoinOrderingRule already ran, add statistics to the dummy table so
+  // that following rules, which expect statistics to be present, do not run into problems.
+  list_as_table->set_table_statistics(TableStatistics::from_table(*list_as_table));
```

Comment:
I don't get what this has to do with the JoinOrderingRule.

Reply:
Re-reading that I wonder if I didn't write too much. Maybe "Add statistics to the dummy table so that following rules, which expect statistics to be present, do not run into problems. " is sufficient?

Reply:
Yep, better.